### DISCUSSION TITLE
del key completely delete note in annotation editor

### DISCUSSION THREAD

**Discussion by xmfbraga** [sumatrapdfreader/sumatrapdf#3872] [Q&A]
Hey everybody!

When testing Sumatra version 3.5.2, I noticed that in the annotation editor, if the del key is pressed while the note is written, the note is completely deleted. This is really bad, because I end up losing everything I had already entered. How can I disable this functionality in Sumatra?

Thanks in advance.

**Comment by kjk** [comment]
It's a bug, I opened https://github.com/sumatrapdfreader/sumatrapdf/issues/3878 to track this
